### DISCUSSION TITLE
Educational Benefits

### DISCUSSION THREAD

**Discussion by jomatheusdev** [community/community#140287] [GitHub Education] [answered]
### Hi everyone,

It's been over 5 days, and I still haven't received my GitHub educational benefit. What can I do?

**Comment by davevad93** [comment, marked as answer]
Hi @jomatheusdev ,

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @jomatheusdev was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by jomatheusdev** [reply]
Thank you so much!!

**Reply by davevad93** [reply]
You're welcome @jomatheusdev 👍

**Comment by angelogalope** [comment]
I think you’ll have to wait for their response or you’ll have to wait for awhile since their are a lot of request they receive and waiting for approval.

**Reply by jomatheusdev** [reply]
Thank you so much!!
